.github/workflows/build.yml

```diff
@@ -381,15 +386,11 @@ jobs:
           cache-control: public,no-cache,proxy-revalidate
 
     - name: upload-kernel-initrd-releases
-      uses: softprops/action-gh-release@v2
       if: ${{ startsWith(github.ref, 'refs/tags/') }}
-      with:
-        draft: true
-        prerelease: true
-        files: |
-          dist/artifacts/harvester*initrd-${{ env.arch }}
-          dist/artifacts/harvester*vmlinuz-${{ env.arch }}
-          dist/artifacts/harvester*images-list-${{ env.arch }}.txt
+      run: |
+        gh release upload ${{ github.ref_name }} dist/artifacts/harvester*initrd-${{ env.arch }}
+        gh release upload ${{ github.ref_name }} dist/artifacts/harvester*vmlinuz-${{ env.arch }}
+        gh release upload ${{ github.ref_name }} dist/artifacts/harvester*images-list-${{ env.arch }}.txt
```

Comment:
It's important to have a draft release created because we want these files attached to the release. Can the gh command create a release?